lib/core/security/tokenRepository.js

```diff
@@ -347,7 +370,7 @@ class TokenRepository extends Repository {
    * @param {string} kuid
    * @returns {Promise}
    */
-  async deleteByKuid (kuid) {
+  async deleteByKuid (kuid, {keepApiKeys = false} = {}) {
```

Comment:
IMHO it should keep the API keys by default because API keys are not supposed to be deleted massively like this